src/big/big_decimal.cr

```diff
@@ -73,10 +73,10 @@ struct BigDecimal
     initialize(num.to_big_i, 0)
   end
 
-  # Creating a `BigDecimal` from `Float` is not supported due to precision loss risks. This call fails at compile time.
+  # Creating a `BigDecimal` from `Float` is not supported due to precision loss risks.
```

Comment:
I would suggest change ctor description here. You **can** create a `BigDecimal` from `Float`, but a warning will be printed.

re: warning, we need a spec that checks for this scenario.